### DISCUSSION TITLE
How to use our own VAD or Speaker Embedding models in speaker diarization?

### DISCUSSION THREAD

**Discussion by yerzhan7orazayev** [NVIDIA-NeMo/NeMo#2075] [Q&A] [answered]
**Describe your question**
Is it possible to use our own VAD or Speaker Embedding models in speaker diarization? If so, how one can do it properly? 
Many thanks.

**Comment by fayejf** [comment, marked as answer]
Thanks for your question. 
Of course! You can change the `diarizer.vad.model_path` and `diarizer.speaker_embeddings.model_path` in [config file](https://github.com/NVIDIA/NeMo/blob/main/examples/speaker_recognition/conf/speaker_diarization.yaml) to use your own models.
